### PR TITLE
[TEST]: PATCH /comments/{commentId}/replies/{replyId}: 

### LINKED ISSUE #183: [TEST]: RepliesEndpointTests - PATCH /comments/{commentId}/replies/{replyId}:
### Describe the solution you'd like
- [ ] Write  `RepliesEndpointTests - PATCH /comments/{commentId}/replies/{replyId}:`

### PR DESCRIPTION
**Pull request by oscar-ekeyekwu**
**Before submitting your PR for review**

- Run `npm run lint` to find errors in code syntax/format
- Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**

Fixes #183 

Summarise the main tasks handled in the pull request

- [x] Added test suite for `PATCH /comments/{commentId}/replies/{replyId} `

Describe (in detail) what the task completed in the pull request does as per the relevant task

**How should this be manually tested?**

**Any background context you want to provide?**

Any pertinent information that should be considered

**What is the link to the issue on Github?**

[Issue #183 ](https://github.com/microapidev/comment-microapi/issues/183)

**Questions:**

If something is unclear or you want some questions to be addressed by your peers, mention them here

### CODE REVIEW

File: __test__/api/comment-vote.test.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ import { describeIfEndpoint } from "../helpers/conditionalTests";
 
 describeIfEndpoint(
   "PATCH",
-  "/comments/:commentId/votes/upvote",
```
Comment (piouson) [inline __test__/api/comment-vote.test.js:10]: Stash your changes if you have any, and then update your forked/local repo

```bash
git stash
git add remote upstream https://github.com/microapidev/comment-microapi.git
git fetch upstream
git merge upstream/develop
```

then you can apply your stash and fix merge conflicts

```bash
git stash pop
```

File: __test__/api/comment-vote.test.js [hunk 2]
```diff
@@ -27,19 +27,12 @@ describeIfEndpoint(
           ownerId: commentObject.ownerId,
         });
       expect(res.status).toBe(200);
-      expect(res.body.data.commentId).toEqual(String(comment._id));
```
Comment (piouson) [inline __test__/api/comment-vote.test.js:30]: same here stash any changes you have and update forked/local repo, then reapply your work.
afterwards you can pop your stash

File: __test__/api/comment-vote.test.js [hunk 3]
```diff
@@ -28,17 +28,37 @@ describeIfEndpoint(
...
+    test("Should reject unknown vote type", async () => {
+      const comment = new CommentModel({
+        content: "this is a comment",
+        ownerId: "useremail3@email.com",
+        origin: "123123",
+        applicationId: global.application._id,
+      });
+      await comment.save();
+
+      const res = await request.patch(`/comments/${comment._id}/votes`).send({
```
Comment (dave-ok) [inline __test__/api/comment-vote.test.js:45]: I thought this was now PATCH /comments/:commentId/votes/upvotes or PATCH /comments/:commentId/votes/downvotes. So there will be no invalid vote anymore. you are either upvoting or downvoting

Comment (oscar-ekeyekwu) [reply __test__/api/comment-vote.test.js:45]: lol, let me remove that then

**Pull request merged**